src/gui/properties/speedplotview.cpp

```diff
@@ -33,15 +33,62 @@
 #include "base/global.h"
 #include "base/utils/misc.h"
 
+namespace
+{
+    enum PeriodInSeconds
+    {
+        MIN1_SEC = 60,
+        MIN5_SEC = 5 * 60,
+        MIN30_SEC = 30 * 60,
+        HOUR6_SEC = 6 * 60 * 60
+    };
+
+    const int MIN5_BUF_SIZE = 5 * 60;
+    const int MIN30_BUF_SIZE = 5 * 60;
+    const int HOUR6_BUF_SIZE = 5 * 60;
+    const int DIVIDER_30MIN = MIN30_SEC / MIN30_BUF_SIZE;
+    const int DIVIDER_6HOUR = HOUR6_SEC / HOUR6_BUF_SIZE;
+};
```

Comment:
looks like this extra `;` is failing lgtm.com

Reply:
deleted